exercises/move_semantics/move_semantics6.rs

```diff
@@ -5,24 +5,22 @@
 // Execute `rustlings hint move_semantics6` or use the `hint` watch subcommand
 // for a hint.
 
-// I AM NOT DONE
-
 fn main() {
     let data = "Rust is great!".to_string();
 
-    get_char(data);
+    let last_char = get_char(&data);
```

Comment:
why did you add a variable definition here?

Reply:
I added it automatically since I thought I would use it in the next steps but found that I didn't need it.  I will remove it